metal3-dev/reset-hosts.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ex
+
+bmo_path=$GOPATH/src/github.com/metal3-io/baremetal-operator
+if [ ! -d $bmo_path ]; then
+    echo "Did not find $bmo_path" 1>&2
+    exit 1
+fi
+
+hosts=$(oc get baremetalhost -n openshift-machine-api -o name | cut -f2 -d/)
+
+(pushd $bmo_path &&
+     go run ./cmd/reset-host/main.go -n openshift-machine-api $hosts)
+
+# NOTE: assumes OS_CLOUD is set correctly
```

Comment:
We could just set it since this will always require OS_CLOUD=metal3 in a dev-scripts environment?

Reply:
I'm not always in a dev-scripts environment these days, but I can set the variable if it isn't already set.